README.md

```diff
@@ -179,7 +179,16 @@ Since Transformers version v4.0.0, we now have a conda channel: `huggingface`.
 conda install -c huggingface transformers
 ```
 
-Follow the installation pages of TensorFlow, PyTorch or Flax to see how to install them with conda. 
+Follow the installation pages of TensorFlow, PyTorch or Flax to see how to install them with conda.
+
+### master version
+
+In some situations where a fix has been applied to the bleeding edge of the code (`master`) and you can't wait for the new release to use it you can install `transformers` directly from `master`, by doing the following:
+
+```
+pip install git+https://github.com/huggingface/transformers
+```
+
```

Comment:
No the README already has a link below to the documentation for the installation from source. We can expand the prose before but I would prefer to leave all specific instructions for a source install on the install doc page.

Reply:
The reason for proposing to have this on the front page (and a very short entry) is to help us devs to quickly link the user to the right quick entry. Otherwise it's much harder to find it to share the link in first place and chances are the users will get lost in the many different options. 

That's why what I added to README.md is just one line of text and one line of code.

Reply:
The install page of the documentation should provide an equally easy link to share.

I really don't want to add too much to the README so it stays concise and to the point, this is why we have an install page in the docs, because the README only has the top-level info.

Reply:
ok, then 

1. may I split up https://huggingface.co/transformers/installation.html#installing-from-source into sub-sections? important to point the user to just the right command and not a whole bunch of them.

2. have to change the anchor text "If you'd like to play with the examples, you must install the library from source." in README.md.

Reply:
Yes, we can split between the two. Not sure what the best names are, maybe editable install and installing from source? The anchor text can be indeed changed in the README to add something like: "If you'd like to play with the examples or need the bleeding edge of the code and can't wait for a new release, you must install the library from source."

Does that sound good to you?

Reply:
That works, thank you, @sgugger - will update it shortly.

Reply:
Done. Please have a look. 

Note that I promoted:
```
pip install git+https://github.com/huggingface/transformers
```
as the main "installing from source" solution, and editable install is actually now what it is.